packages/react/src/__tests__/ReactProfilerDOM-test.internal.js

```diff
@@ -18,10 +18,7 @@ let ReactCache;
 function initEnvForAsyncTesting() {
   // Boilerplate copied from ReactDOMRoot-test
   // TODO pull this into helper method, reduce repetition.
-  const originalDateNow = Date.now;
-  global.Date.now = function() {
```

Comment:
Lolex will fake this.

The test also passed when removing it when using Jest 23, not sure what it was supposed to do?

Reply:
This is curious! I'm not sure what this was for. I'm guessing that this was just an artifact copy/pasting a snippet from ReactDOMRoot-test. It's quite a bit more sophisticated:

https://github.com/facebook/react/blob/c84b9bf828c38e0b24042ebb9cd859292146a6fb/packages/react-dom/src/__tests__/ReactDOMRoot-test.js#L23-L45

I think this is safe to remove.